scripts/configure-hypervisor.sh

```diff
@@ -304,7 +304,7 @@ generate_qemu_options() {
 	# implicitly enabled in Fedora 27).
 	qemu_options+=(size:--disable-linux-aio)
 
-	if [ "${qemu_version_major}" -ge 3 ] && [ "${qemu_version_minor}" -ge 1 ]; then
+	if [[ "${qemu_version_major}" -ge 4 || ( "${qemu_version_major}" -ge 3  &&  "${qemu_version_minor}" -ge 1 ) ]]; then
```

Comment:
This now looks a bit wonky - should that v3.1 check be more like `major -eq 3`, and not `-ge` ?